### PR TITLE
Add prior box clustered

### PR DESCRIPTION
**Pull request by XinWangIntel**
depends on https://github.com/plaidml/openvino/pull/133, add PriorBoxClusteredOp.

### CODE REVIEW

**Comment by YangleiZouIntel** [comment]
CI should pass with https://github.com/plaidml/plaidml/pull/1687 merged.

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    auto input1_shape = cast_constant_operand<int64_t>(1, layer);
+    int64_t layer_height = input0_shape[0];
+    int64_t layer_width = input0_shape[1];
+    int64_t img_height = input1_shape[0];
+    int64_t img_width = input1_shape[1];
+
+    float step_w = attrs.step_widths;
+    float step_h = attrs.step_heights;
+
+    // TODO: Ref implementation in openvino currently not open "img_h && img_w && step" attributes
```
Comment (tzerrell) [inline plaidml/bridge/openvino/ops/prior_box_clustered.cpp:42]: Thanks for noting the work that remains outstanding here.

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    edsl::Tensor BW = edsl::Constant(buffer_bw, "bw") / 2.0f / img_width;
+
+    Buffer buffer_bh(shape_box);
+    buffer_bh.copy_from(attrs.heights.data());
+    edsl::Tensor BH = edsl::Constant(buffer_bh, "bh") / 2.0f / img_height;
+
+    edsl::Tensor C_out = CW_normalized * CW_mask + CH_normalized * CH_mask + BW * BW_mask + BH * BH_mask;
+    if (attrs.clip) C_out = op::clip(C_out, edsl::Tensor(0.0f), edsl::Tensor(1.0f));
+
+    // Use varaince as another channel
```
Comment (tzerrell) [inline plaidml/bridge/openvino/ops/prior_box_clustered.cpp:101]: Typo: `varaince` -> `variance`

**Review by tzerrell — COMMENTED** [review]
Looks good, minor comments only

File: plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/prior_box_clustered.cpp [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    0.0f,
+    2.0f,
+};
+
+const std::vector<float> step_heights = {
+    0.0f,
+    1.5f,
+};
+
+const std::vector<float> offsets = {0.5f};
```
[the remark this answers is not in the document]
Comment (XinWangIntel) [reply plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/prior_box_clustered.cpp]: Done, add 0.0f

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp (hunk 2 above)
Comment (XinWangIntel) [reply plaidml/bridge/openvino/ops/prior_box_clustered.cpp]: Done

**Comment by tzerrell** [comment]
Updated name since it will be squashed commit message

**Review by tzerrell — APPROVED** [review]

**Pull request merged**